actions/installer/src/index.ts

```diff
@@ -0,0 +1,141 @@
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+import * as io from "@actions/io";
+import * as tc from "@actions/tool-cache";
+import * as crypto from "crypto";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+const LATEST_VERSION = "v1.3.0";
+
+const VERIFIER_V130_SHA256 =
+  "1326430d044e8a9522c51e5f721e237b5f75acb6b4e518d129f669403cf7a79a";
+const BINARY_NAME = "slsa-verifier";
+const PROVENANCE_NAME = "slsa-verifier-linux-amd64.intoto.jsonl";
+
+// If true, the input string conforms to slsa-verifier's versioning system.
+export function validVersion(version: string): boolean {
+  const re = /(v[0-9]+\.[0-9]+\.[0-9]+|latest)/;
+  return re.test(version);
+}
+
+// If true, then the file in `path` has the same SHA256 hash as `expectedSha256Hash``.
+export function fileHasExpectedSha256Hash(
+  filePath: string,
+  expectedSha256Hash: string
+): boolean {
+  if (!fs.existsSync(filePath)) {
+    throw new Error(`File not found: ${filePath}`);
+  }
+  const untrustedFile = fs.readFileSync(filePath);
+  const computedSha256Hash = crypto
+    .createHash("sha256")
+    .update(untrustedFile)
+    .digest("hex");
+  return computedSha256Hash === expectedSha256Hash;
+}
+
+let tmpDir: string;
+
+// Delete bootstrap version and maybe installed version
+async function cleanup(): Promise<void> {
+  await io.rmRF(`${tmpDir}`);
+}
+
+async function run(): Promise<void> {
+  // Get requested verifier version and validate
+  const version = process.env.GITHUB_ACTION_REF || LATEST_VERSION;
+  if (!validVersion(version)) {
+    core.setFailed(
+      "Invalid version provided. For the set of valid versions, see https://github.com/slsa-framework/slsa-verifier/releases."
+    );
+    return;
+  }
+
+  tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), "slsa-verifier_"));
+  const bootstrapDir = `${tmpDir}/bootstrap`;
+  const installDir = `${tmpDir}/${version}`;
+
+  let bootstrapVerifierPath;
+  try {
+    // Download bootstrap version and validate SHA256 checksum
+    bootstrapVerifierPath = await tc.downloadTool(
+      "https://github.com/slsa-framework/slsa-verifier/releases/download/v1.3.0/slsa-verifier-linux-amd64",
```

Comment:
nit: use LATEST_VERSION in the url here. Or create another global that sets the bootstrap URL or bootstrap version.